src/rpc/util.h

```diff
@@ -43,24 +43,54 @@ struct RPCArg {
     const Type m_type;
     const std::vector<RPCArg> m_inner; //!< Only used for arrays or dicts
     const bool m_optional;
```

Comment:
Sure, but this will be a follow up pull request, since it needs someone to actually supply the default value when the argument is optional. See also the comment in the cpp file:

```cpp
// TODO enable this assert, when all optional parameters have their default value documented
//assert(false);